tox.ini

```diff
@@ -1,28 +0,0 @@
-# Tox (https://tox.readthedocs.io/) is a tool for running tests
-# in multiple virtualenvs. This configuration file will run the
-# test suite on all supported python versions. To use it, "pip install tox"
-# and then run "tox" from this directory.
-# 
-# To run different environments, try `tox -e py27` or `tox -e docs`
-# 
-
-[tox]
-envlist = py36, py37, py38, py39
-
-[gh-actions]
-python =
-    3.6: py36
-    3.7: py37
-    3.8: py38
-    3.9: py39
-
-[testenv]
-commands =
-  pip install -e .
-  {envpython} setup.py test
```

Comment:
this is a convenient way to do this, and also decouples the source code repository from the github ci, and i feel like it should be self contained here?

Reply:
I'm going to restore the `tox.ini` file